docs/networking/native-ipv6-networking.md

```diff
@@ -92,27 +95,48 @@ While default IPv6 addresses are configured automatically, you will need to stat
       auto lo
       iface lo inet loopback
 
-      # The primary network interface (IPv4)
+      # Automatically brings up the default network interface (eth0)
       auto eth0
-      iface eth0 inet dhcp
+      allow-hotplug eth0
 
       # IPv6 Address Blocks
-      # You should add an additional block for each IPv6 address you need configured.
-      # You do not need to configure a block for your default IPv6 address, it will be brought up via SLAAC
+      # Add a static block for your default public IPv6 address, and include a directive for its gateway.
+
+      iface eth0 inet6 static
+        address 2001:db8:a0b:12f0::1/64
+
+      # Add an additional block for each IPv6 address you need to configure.
+      
       iface eth0 inet6 static
-        address 2001:DB8:2000:aff0::1/64
+        address 2001:db8:2000:aff0::1/64
 
       iface eth0 inet6 static
-        address 2001:DB8:2000:aff0::2/64
+        address 2001:db8:2000:aff0::2/64
       ~~~
 
-2.  Restart networking. As this will break an SSH connection, this command should be performed in [LISH](/docs/networking/using-the-linode-shell-lish)
+        {: .note}
+        >On Debian Jessie, your default IPv6 address provided by SLAAC will no longer be automatically assigned after you request a /64 pool. You will need to manually add it as a static address or IPv6 routing will not work.
+
+2.  For /56 and /64 pools, addresses within your pool will be routed to your Linode's default IP address, or another Linode on your account in the same datacenter. You will see where the pool is routed under "Public IP Pools" within the Linode Manager's Remote Access tab. You must enable packet forwarding on that Linode to allow it to act as a router and enable external traffic from addresses within your IPv6 pool:
+
+    {: .file}
+    /etc/sysctl.conf
+    : ~~~
+      # Uncomment the next line to enable packet forwarding for IPv6
+      #  Enabling this option disables Stateless Address Autoconfiguration
+      #  based on Router Advertisements for this host
+      net.ipv6.conf.all.forwarding=1
+      ~~~
+
+    For addresses within a /116 pool, the above change is not necessary as routing will be automatically configured. Addresses in a /116 pool are routed to other Linodes on your account within the same datacenter.
+
+3.  Restart networking. This command should be performed in [Lish](/docs/networking/using-the-linode-shell-lish), as it will terminate an SSH connection.
 
         ifdown -a && ifup -a
 
 ### CentOS/Fedora
```

Comment:
Since you have a section for Fedora21, this section should indicate which versions of Fedora this applies to. Same on Line 174.